packages/wallet/src/services/DelegationTracker/RewardAccounts.ts

```diff
@@ -177,8 +173,10 @@ const accountCertificateTransactions = (
       transactions
         .map(({ tx, epoch }) => ({
           certificates: (tx.body.certificates || [])
-            .filter((cert): cert is Cardano.StakeDelegationCertificate | Cardano.StakeAddressCertificate =>
-              [...RegAndDeregCertificateTypes, Cardano.CertificateType.StakeDelegation].includes(cert.__typename)
+            .filter((cert): cert is Cardano.RegAndDeregCertificateUnion | Cardano.StakeDelegationCertificateUnion =>
```

Comment:
Here it seems `hasDelegationCert` could be used; since we already have this helper, better to remove code duplication.